### PR TITLE
fix: Removed cardView, replaced with shapes.

### LINKED ISSUE #1455: Incorrectly rounded views
**Describe the bug**

In many fragments eg. fragment_event cardviews are used to give rounded corners to buttons, image views, etc. This is a workaround and not the correct way to make rounded views.

**To Reproduce**

Check fragment_event.xml

**Expected behavior**

xml files with correct shape should be created and set as background for views which require rounded corners.

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #1455 

Changes: Removed unnecessary cardViews  and replaced them by using shapes and setting them as background for views. Had to use cliptooutline in code because cliptooutline in xml has some bug. Check https://issuetracker.google.com/issues/37036728 for more details. Layouts look the same.

### CODE REVIEW

File: app/src/main/res/layout/fragment_event.xml [hunk 1]
```diff
@@ -15,28 +15,22 @@
         android:layout_height="match_parent"
         app:layout_dodgeInsetEdges="bottom">
 
-        <androidx.cardview.widget.CardView
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
+        <TextView
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_event.xml]: Use Button

**Comment by addiegupta** [comment]
@angmas1 @iamareebjamal  I have converted the CardView into a MaterialButton in #1021

File: app/src/main/res/layout/fragment_event.xml (hunk 1 above)
Comment (angmas1) [reply app/src/main/res/layout/fragment_event.xml]: done

**Comment by addiegupta** [comment]
@angmas1 sorry for the confusion. I have converted this exact same button to a MaterialButton in #1021 hence there is a merge conflict

**Comment by angmas1** [comment]
> @angmas1 sorry for the confusion. I have converted this exact same button to a MaterialButton in #1021 hence there is a merge conflict

Okay, I will resolve it and let your code stay.

**Pull request merged**